### PR TITLE
Update tests to most recent alpha5 dependencies

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

File: testing/environments/docker/elasticsearch/Dockerfile-5.0.0-alpha4 [hunk 1]
```diff
@@ -5,7 +5,7 @@ ENV ELASTICSEARCH_VERSION master
 
 
 COPY setup.sh /
-RUN bash setup.sh http://download.elastic.co/elasticsearch/staging/5.0.0-alpha4-b0da471/org/elasticsearch/distribution/deb/elasticsearch/5.0.0-alpha4/elasticsearch-5.0.0-alpha4.deb
+RUN bash setup.sh http://download.elastic.co/elasticsearch/staging/5.0.0-alpha5-95ae58b/org/elasticsearch/distribution/deb/elasticsearch/5.0.0-alpha5/elasticsearch-5.0.0-alpha5.deb
```
Comment (tsg) [inline testing/environments/docker/elasticsearch/Dockerfile-5.0.0-alpha4]: The file is still called alpha4, you probably need to rename it to alpha5, right?

File: testing/environments/5.0.0-alpha4.yml [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
 
 logstash:
   build: ./docker/logstash
-  dockerfile: Dockerfile-5.0.0-alpha4
+  dockerfile: Dockerfile-5.0.0-alpha5
 
 kibana:
   build: ./docker/kibana
-  dockerfile: Dockerfile-5.0.0-alpha4
+  dockerfile: Dockerfile-5.0.0-alpha5
```
Comment (tsg) [inline testing/environments/5.0.0-alpha4.yml]: Also this file is named alpha4 still + I don't see the kibana Dockerfile for alpha5.

File: testing/environments/docker/elasticsearch/Dockerfile-5.0.0-alpha4 (hunk 1 above)
Comment (ruflin) [inline testing/environments/docker/elasticsearch/Dockerfile-5.0.0-alpha4]: I initially added alpha5 files but revert it as not all alpha5 builds are available yet, so I only updated the snapshots. Forgot to revert this one :-(

File: testing/environments/5.0.0-alpha4.yml (hunk 2 above)
Comment (ruflin) [inline testing/environments/5.0.0-alpha4.yml]: Oh, I just realised I didn't push my most recent change locally as I forgot to use force -> see also comment below

**Pull request merged**